src/lightning_app/core/app.py

```diff
@@ -70,7 +70,7 @@ def __init__(
             root: The root ``LightningFlow`` or ``LightningWork`` component, that defines all the app's nested
                  components, running infinitely. It must define a `run()` method that the app can call.
             flow_cloud_compute: The default Cloud Compute used for flow, Rest API and frontend's.
-            debug: Whether to activate the Lightning Logger debug mode.
+            log_level: Whether to activate the Lightning Logger debug mode.
```

Comment:
Change log string to something like “The log level for the app, one of [‘info’, ‘debug’]”

Reply:
Done !